packages/vscode-boxel-tools/src/file-system-provider.ts

```diff
@@ -359,15 +360,12 @@ export class RealmFS implements vscode.FileSystemProvider {
 
     try {
       let headers = {
-        Accept: 'application/vnd.api+json',
+        Accept: SupportedMimeType.JSONAPI,
         Authorization: `${await this.realmAuth.getJWT(apiUrl)}`,
       };
 
-      // Add special header for .gts files
-      if (uri.path.endsWith('.gts')) {
-        headers['Accept'] = 'application/vnd.card+source';
-      } else if (uri.path.endsWith('.json')) {
-        headers['Accept'] = 'application/vnd.card+json';
+      if (hasExecutableExtension(uri.path)) {
```

Comment:
shouldn't card-source accept header be used for all file types, like the monaco editor in the host app?